build/Makefile

```
@@ -0,0 +1,167 @@
+SHELL:=/usr/bin/env bash
+SRC_DIR:=../src
+SRC_FILES:=$(shell find $(SRC_DIR) -name '*.xml' -o -name '*.json')
+SRC_XML_PROFILES:=$(shell find $(SRC_DIR) -name '*profile.xml')
+SRC_READMES:=$(shell find $(SRC_DIR) -iname 'README.md')
+GEN_CONTENT_DIR:=generated
+GEN_READMES:=$(patsubst $(SRC_DIR)/%,$(GEN_CONTENT_DIR)/%,$(SRC_READMES))
+GEN_XML_FILES:=$(patsubst $(SRC_DIR)/%,$(GEN_CONTENT_DIR)/%,$(SRC_FILES))
+GEN_XML_RESOLVED_CATALOGS:=$(subst _profile.xml,-resolved-profile_catalog.xml,$(patsubst $(SRC_DIR)/%,$(GEN_CONTENT_DIR)/%,$(SRC_XML_PROFILES)))
+GEN_JSON_FILES:=$(subst xml,json,$(GEN_XML_FILES))
+GEN_MIN_JSON_FILES:=$(subst .json,-min.json,$(subst xml,json,$(GEN_XML_FILES) $(GEN_XML_RESOLVED_CATALOGS)))
+GEN_YAML_FILES:=$(subst xml,yaml,$(GEN_XML_FILES) $(GEN_XML_RESOLVED_CATALOGS))
+CURL_INSTALL_OPTS:=--silent --location
+XMLLINT_PATH=$(shell which xmllint || { echo "Use operating system to install XMLLINT_INSTALL_COMMAND"; exit 1; })
+NPM_PREFIX_DIR:=oscal/build
+NPM_PKGS_DIR:=node_modules
+XSLT_RUNNER:=oscal/build/xslt-runner.sh
+PROFILE_RESOLVER_RUNNER:=oscal/src/utils/resolver-pipeline/oscal-profile-resolve.sh
+PROFILE_RESOLVER_ARGS:="uuid-method='random-xslt'"
+XML_JSON_CONVERTER_XSLT:=oscal/build/generated/oscal_complete_xml-to-json-converter.xsl
+OSCAL_COMPLETE_XML_SCHEMA:=oscal/build/generated/oscal_complete_schema.xsd
+OSCAL_COMPLETE_JSON_SCHEMA:=oscal/build/generated/oscal_complete_schema.json
+
+.PHONY: help
+# Run "make" or "make help" to get a list of user targets
+# Adapted from https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+help: ## Show this help message
+	@grep -E '^[a-zA-Z_-]+:.*?##.*$$' $(MAKEFILE_LIST) | awk 'BEGIN { \
+	 FS = ":.*?## "; \
+	 printf "\033[1m%-30s\033[0m %s\n", "TARGET", "DESCRIPTION" \
+	} \
+	{ printf "\033[32m%-30s\033[0m %s\n", $$1, $$2 }'
+
+.PHONY: all
+all: build dependencies artifacts checks ## Run all steps for content preparation
+
+.PHONY: build
+build: ## Build core OSCAL artifacts to convert content examples
+	$(MAKE) -C oscal/build dependencies
+	$(MAKE) -C oscal/build artifacts
+
+# Used to automatically install certain executables
+JQ_INSTALL_BIN:=jq-linux-amd64
+JQ_INSTALL_VERSION:=jq-1.7
+JQ_INSTALL_URL:=https://github.com/jqlang/jq/releases/download/$(JQ_INSTALL_VERSION)/$(JQ_INSTALL_BIN)
+JQ_INSTALL_PATH:=./jq
+JQ_PATH:=$(shell which jq || echo $(JQ_INSTALL_PATH))
+JQ_INSTALL_COMMAND=curl $(CURL_INSTALL_OPTS) -o $(JQ_INSTALL_PATH) $(JQ_INSTALL_URL) && chmod +x $(JQ_INSTALL_PATH)
+
+$(JQ_INSTALL_PATH):
+	@$(JQ_INSTALL_COMMAND)
+
+YQ_INSTALL_BIN:=yq_linux_amd64
+YQ_INSTALL_VERSION:=v4.35.2
+YQ_INSTALL_URL:=https://github.com/mikefarah/yq/releases/download/$(YQ_INSTALL_VERSION)/$(YQ_INSTALL_BIN)
+YQ_INSTALL_PATH:=./yq
+YQ_PATH:=$(shell which yq || echo $(YQ_INSTALL_PATH))
+YQ_INSTALL_COMMAND:=curl $(CURL_INSTALL_OPTS) -o $(YQ_INSTALL_PATH) $(YQ_INSTALL_URL) && chmod +x $(YQ_INSTALL_PATH)
+
+$(YQ_PATH):
```

B: Correct behavior to make manual invocation possible on machines with `yq` installed:
```suggestion
$(YQ_INSTALL_PATH):
```

---

Corrected in https://github.com/usnistgov/oscal-content/commit/99669b4bd2603f0eea88a5d8a2ca2df76921b5d6.